build_deploy.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# --------------------------------------------
+# Export vars for helper scripts to use
+# --------------------------------------------
+# name of app-sre "application" folder this component lives in; needs to match for quay
+export COMPONENT="content-sources"
```

Comment:
I think its supposed to be:

```
APP_NAME="content-sources"  # name of app-sre "application" folder this component lives in
COMPONENT_NAME="content-sources-frontend"  # name of app-sre "resourceTemplate" in deploy.yaml for this 
```

Reply:
I'm going to disagree with myself, after seeing Ondrej's work on provisioning-frontend.